src/nexusGrades.js

```diff
@@ -87,6 +92,38 @@ fluid.nexus.writeDefaults.handleRequest = function (gradeName, request) {
     request.events.onSuccess.fire();
 };
 
+fluid.defaults("fluid.nexus.readComponent.handler", {
+    gradeNames: ["kettle.request.http"],
+    invokers: {
+        handleRequest: {
+            funcName: "fluid.nexus.readComponent.handleRequest",
+            args: ["{request}.req.params.path", "{request}", "{fluid.nexus}.nexusComponentRoot"]
+        }
+    }
+});
+
+// TODO: this API endpoint should really supply the potentia and model of a component,
+//       i.e. serialized material that could be used to reconstruct the component in its current state.
+//       This is not straightforward to provide under the current version of Infusion.
```

Comment:
It is actually relatively straightforward, it's just that we might not be ready to do it yet and/or the result might not be entirely intelligible to the user without some supporting documentation on what a potentia actually is, etc : P